src/java/org/apache/cassandra/service/reads/ReadCallback.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.cassandra.config.DatabaseDescriptor;
 import org.apache.cassandra.db.MessageParams;
 import org.apache.cassandra.locator.ReplicaPlan;
+import org.apache.cassandra.transport.Dispatcher;
 import org.apache.cassandra.utils.concurrent.Condition;
```

Comment:
nit: These imports are all out of place?

Reply:
Must have been merge. Thank you for spotting!